src/libraries/Microsoft.Extensions.Logging.EventSource/src/LoggingEventSource.cs

```diff
@@ -329,7 +329,7 @@ private void FireChangeToken()
         [NonEvent]
         private static LoggerFilterRule[] ParseFilterSpec(string filterSpec, LogLevel defaultLevel)
         {
-            if (filterSpec == string.Empty)
+            if (filterSpec?.Length == 0)
```

Comment:
Your benchmarking was about `s == string.Empty` vs `s.Length == 0`... but this check is `s != null && s.Length == 0`.  Did you measure that?

Regardless, in this particular case, there's no chance the difference between any of these versions is measurable.  We should leave it the way it was rather than churning this code.

Reply:
@stephentoub Ok, I'll revert the changes where I used `s?.Length == `.